src/transport/HttpClient.ts

```diff
@@ -331,6 +334,10 @@ class StubHttpClient implements HttpClientBase {
     getBoundedQuizzes(noteId: string): Promise<any> {
         return Promise.resolve(undefined);
     }
+
+    updateResourcePermissionLevel(resourceType: ResourceType, resourceId: string, value: AccessType): Promise<void> {
+        return Promise.resolve(undefined);
```

Comment:
should it has undifined in?

Reply:
this is stub implementation
Maybe we could remove that class since we don't use it

Reply:
choose what you think is better